openedx/features/pakx/lms/pakx_admin_app/views.py

```diff
@@ -18,9 +21,27 @@ class UserProfileViewSet(viewsets.ModelViewSet):
     pagination_class = PakxAdminAppPagination
     serializer_class = UserSerializer
     filter_backends = [OrderingFilter]
-    OrderingFilter.ordering_fields = ('id', 'name', 'email', 'employee_id')
     ordering = ['-id']
 
+    def create(self, request, *args, **kwargs):
+        if request.data.get("username") and request.data.get("email") and request.data.get(
+                "employee_id") and request.data.get("language"):
+            return self.create_new_user(request)
+
+        return Response(status=status.HTTP_400_BAD_REQUEST)
```

Comment:
please return a detail response at field level, i.e this is field is required instead of just a general 400 error.